src/main/java/com/elytradev/concrete/inventory/gui/ConcreteContainer.java

```diff
@@ -232,23 +234,22 @@ public ItemStack transferToInventory(ItemStack stack, IInventory inventory) {
 	public boolean canStackTogether(ItemStack src, ItemStack dest) {
 		if (src.isEmpty() || dest.isEmpty()) return false; //Don't stack using itemstack counts if one or the other is empty.
 
+		NBTTagCompound srcTag = src.getTagCompound();
```

Comment:
There's a method builtin to forge, ItemHandlerHelper.canItemStacksStack I believe.